src/app/item-review/destiny-tracker.service.ts

```diff
@@ -199,6 +207,10 @@ export function DestinyTrackerService(
       return _d2bulkFetcher.getCache();
     },
 
+    getReviewMode(defs: D2ManifestDefinitions, review: DtrUserReview): string {
+      return getReviewMode(defs, review);
+    },
```

Comment:
You can write this as `getReviewMode,` since you're just calling the function with the same name and same arguments.